### DISCUSSION TITLE
Visual Code Server - Live Preview Extension

### DISCUSSION THREAD

**Discussion by DyloFi** [microsoft/vscode-discussions#279] [Extension Development QnA] [answered]
Hey, sorry to bother guys. I was just wondering if it’s possible to get the extension “Live Preview” working with VS Code Server?

**Comment by Talm009** [comment]
I have the same problem.

**Comment by gjsjohnmurray** [comment, marked as answer]
I recommend you search existing issues at https://github.com/microsoft/vscode-livepreview/issues and open a new one if necessary.
